### ISSUE TITLE
tkn version gives unknown for Pipeline version

### ISSUE THREAD

**Issue by savitaashture** [tektoncd/cli#845]
# Version and Operating System

**tkn Version:**
Built from master

**Operating System:** 
Linux
# Expected Behavior
If Pipeline installed from master branch
```
./tkn version
Client version: dev
Pipeline version: devel

```
# Actual Behavior
```
./tkn version
Client version: dev
Pipeline version: unknown
```
# Steps to Reproduce the Problem

1. Install tkn
2. Install Pipeline
3. tkn version

# Additional Info

**Comment by danielhelfand** [comment]
@savitaashture Was this fixed by #847? Can this be closed?

**Comment by vdemeester** [comment]
yeah :)
/close
